src/main/java/frc/robot/commands/SpinControlPanel.java

```diff
@@ -25,6 +25,6 @@ public SpinControlPanel(ControlPanelSubsystem controlPanelSubsystem) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    controlPanelSubsystem.setMotorSpeed(.5);
+    controlPanelSubsystem.setMotorSpeed(1);
```

Comment:
Let's use a constant here.

```suggestion
    controlPanelSubsystem.setMotorSpeed(ControlPanelConstants.MOTOR_SPEED);
```

or something